src/gui/main.cpp

```diff
@@ -53,9 +53,10 @@ int main(int argc, char **argv)
     // want to use sizes relative to the font size everywhere.
     // This is automatic on OS X, but opt-in on Windows and Linux
     // https://doc-snapshots.qt.io/qt5-5.6/highdpi.html#qt-support
-#if QT_VERSION < QT_VERSION_CHECK(5, 6, 0)
-    qputenv("QT_DEVICE_PIXEL_RATIO", "auto");
-#else
+    //
+    // Note: QT_DEVICE_PIXEL_RATIO = auto for Qt < 5.6.0 is broken,
```

Comment:
You don't need this line since you already wrote it in the commit message and the comment isn't useful without that code. The comment also only applies to Linux.